apa-annotated-bibliography.csl

```diff
@@ -51,11 +51,21 @@
   <locale xml:lang="de">
     <terms>
       <term name="et-al">et al.</term>
+      <term name="letter">persönliche Kommunikation</term>
+      <term name="letter" form="short">Brief</term>
     </terms>
   </locale>
   <locale xml:lang="es">
     <terms>
       <term name="from">de</term>
+      <term name="letter">persönliche Kommunikation</term>
```

Comment:
that's not right ;)

Reply:
lol. Oops. My copy paste skills are amazing.